spec/rmagick/kernel_info/initialize_spec.rb

```diff
@@ -3,11 +3,7 @@
     Magick::KernelInfoType.values do |kernel|
       k = kernel.to_s.sub('Kernel', '')
 
-      if kernel == Magick::UserDefinedKernel
-        expect { described_class.new(k) }.to raise_error(RuntimeError)
-      else
-        expect(described_class.new(k)).to be_instance_of(described_class)
-      end
+      expect(described_class.new(k)).to be_instance_of(described_class) unless [Magick::UserDefinedKernel, Magick::UndefinedKernel].include?(kernel)
```

Comment:
I think what we could probably do is add a guard clause to the top:

```rb
next if [Magick::UserDefinedKernel, Magick::UndefinedKernel].include?(kernel)

k = kernel.to_s.sub('Kernel', '')

expect(described_class.new(k)).to be_instance_of(described_class)
```

Reply:
Yeah that looks nicer, pushed a patch for that.